### ISSUE TITLE
Stream Connectors Stalling when using Load Balancing or Container Orchestration

### ISSUE THREAD

**Issue by MaxwellDPS** [OpenCTI-Platform/client-python#306]
## Description

It seems that when the OpenCTI API sits behind a load balancer if the session for a stream connector hops destinations the container seems to stall.
I can turn on session affinity, but this would defeat the purpose (of having multiple replicas, when they dont distribute the load to under-worked containers) in the case of any non-stream connection.

The code that manages the stream connection should have better keep-alive abilities with the ability to hop sessions given this is a supported method of running OpenCTI. 
It is especially problematic if say you were feeding a KV store and the connector is loosing its connection ~ every 15 min leading to a restart and backoff (Kubernetes waits to restart to not overload the host).

This does seem to be affecting the events received, but I am working on confirming that events are lost

```
NAME                                READY   STATUS     RESTARTS        AGE
stream-connector-example            2/2     Running    9 (3m51s ago)   118m
```

## Environment

1. OS (where OpenCTI server runs): CentOS Stream
2. OpenCTI version: 5.4.1
3. Other environment details: Kubernetes Deployment; Multiple replicas of the API load balanced by Kubernetes

## Reproducible Steps

Steps to create the smallest reproducible scenario:
1. Deploy to an env where (Connector[stream]) <----> (Load balancer) <-----> (Multiple replicas of the api)
2. Wait for LB to change backend 
3. Container will stall and give error below

## Expected Output

TLDR; The code that manages the stream connection should have better keep-alive abilities with the ability to hop sessions given this is a supported method of running OpenCTI. 

## Actual Output

```
INFO:root:no healthy upstream
ERROR:root:Error pinging the API
ERROR:root:Time since last heartbeat exceeded 45s, stopping the connector
NoneType: None
Killed
```

## Additional information

If there is anything I can do to help test, I would be more than happy :)

**Comment by SamuelHassine** [comment]
Hello @MaxwellDPS,

With the latest enhancements in both streams and Python library, this should be definitely solved from 5.7.0. Please re-open if you still encounter problems.

Kind regards,
Samuel
